sql/analyzer/load_events.go

```diff
@@ -38,6 +38,14 @@ func loadEvents(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, sel Rul
 			}
 			newShowEvents.Events = loadedEvents
 			return &newShowEvents, transform.NewTree, nil
+		case *plan.AlterEvent:
+			newAlterEvents := *node
```

Comment:
(super, super minor nit) `newAlterEvent` (no `s`) might be a little more accurate since only one event can be altered with the statement. 😄